### ISSUE TITLE
Mobile Discord incorrect emote size/Animated emotes aren't animated 

### ISSUE THREAD

**Issue by AkiraStein** [riolubruh/YABDP4Nitro#236]
Animated emotes still aren't actually animated on both (?) versions of Discord, it's a still image. I'm using 5.5.7

The option to upload emotes as images fixes this, however that makes emotes appear too big on mobile (this wasn't even a thing before some recentish discord updates, now if you upload your own gif it's gonna upscale even if it's small).

I also noticed that if the link to emote (gif) doesn't contain "?" and anything after that, that is also a solution, with a compromise of emote not being the right size, but neither is it too big. 
Ex: 
https://cdn.discordapp.com/emojis/1210767163242057798.gif?size=48&1 - would either not animate or appear too big (depending if the option is checked)
https://cdn.discordapp.com/emojis/1210767163242057798.gif - will animate in any scenario, but the size is a bit bigger than max emote size (around 130 pixels)

Also, if you add a "big" gif into favorite gifs and use it from there, the size on mobile is fixed again. 
I'm not sure how it works, but it seems to do something with the link again because a link from favorites looks something like this (not 100% sure):
https://cdn.discordapp.com/attachments/767997937266982914/1307230134423519232/huhExplode.gif
or
https://cdn.discordapp.com/attachments/767997937266982914/1307231068293955665/DonoWall.gif?ex=67398d4d&is=67383bcd&hm=4504cd8bd7d12c22fc3dbdca0cdccae14d76d84bc93b5ab8d46b20945d0c4c9a&

ALSO 
There's this type of link which idr where i got from, i think aliucord's nitro spoof plugin?
https://cdn.discordapp.com/emojis/1199533429327417464.gif?size=48&animated=true&name=catSlap
This one is animated on mobile (i checked on unmodded), but not on desktop

tl;dr
If you can't fix this, maybe add a checkbox that would make links not use the "?size=" thing?
Or an option that shows the link to emote in the message box before you send it, so it's possible to edit it manually?

Image:
1. Not Animated
2. Animated with checkbox on
3. Animated through editing the link
![8770c48c-dd43-4af5-bfd1-9075b5dd9fe3](https://github.com/user-attachments/assets/02a6fe71-ac70-4b96-8c10-ccc94fad2c2f)

BTW They're still animated in the selection box and message box

**Comment by riolubruh** [comment]
Mobile Discord is a little stupid nowadays in that it'll make uploaded files appear larger for some reason even though that doesn't happen with linked images. I can't fix that, but them not animating is a separate, albeit mobile-related issue which will be handled in #235.
